src/battle.py

```diff
@@ -743,26 +894,62 @@ def handle_projectiles(self, owner_nation, proj, enemy, enemy_force, stats, enem
 
                             #Allow the troop being hit by the projectile to try and defend a little.
                             defense = p.target.get_ranged_defense(owner_nation.tech.get_best_in_category('material'))
+                            is_shield = p.target.weapons[1].shield 
```

Comment:
Not necessarily important for this pull request, but probably will want to add some graphical indication that a unit has a shield.

Reply:
I'm planning on doing so.